narwhal/storage/src/node_store.rs

```diff
@@ -120,11 +106,6 @@ impl NodeStorage {
             // This can be removed when DBMap supports removing tables.
             _sub_dag_index_map,
             committed_sub_dag_map,
-            processed_messages_map,
-            used_messages_map,
-            confirmations_map,
-            dkg_output_map,
-            randomness_round_map,
```

Comment:
@aschran can we confirm that this is ok to do now? As far as I remember we can't simply delete column families (ex see that `_sub_dag_index_map` case above) when the db has been already opened for existing column families.

Reply:
Good call.

Reply:
I think we thought about changing the rocksdb open logic across Sui and Narwhal to tolerate the issue, but decided against it for a reason.